stable/insights/values.yaml

```diff
@@ -1,39 +1,49 @@
 grafana:
   enabled: true
   image:
-    tag: 7.5.4
-  # For complete list of configuration options for  
+    tag: 9.5.6
+  # For complete list of configuration options for
   # grafana see the documentation @ https://github.com/grafana/helm-charts/tree/main/charts/grafana.
   # sidecar setup here is for custom dashboards and datasource for nuocd pushing metrics
   # to influx
   sidecar:
     dashboards:
       enabled: true
-      label: 'grafana_dashboard'
+      label: "grafana_dashboard"
       searchNamespace: null
       folder: "/var/lib/grafana/dashboards"
       provider:
-        name:           "nuodb"
+        name: "nuodb"
         allowUiUpdates: true
     datasources:
       enabled: true
-      label: 'grafana_datasource'
+      label: "grafana_datasource"
       searchNamespace: null
 
-influxdb:
+influxdb2:
   enabled: true
   image:
-    tag: 1.8
-  # For complete list of configuration options for  
+    tag: 2.7
+  adminUser:
+    password: "helloworld"
+    bucket: "telegraf"
+    token: "quickbrownfoxjumpsoverlazydog"
```

Comment:
You may want to document how to generate unique values for the `token` and `password` in the `stable/insights/README.md`.